decidim-core/spec/helpers/decidim/widget_urls_helper_spec.rb

```diff
@@ -4,6 +4,14 @@
 
 module Decidim
   describe WidgetUrlsHelper do
+    let(:redesign_enabled) { false }
+
+    before do
+      # rubocop:disable RSpec/AnyInstance
+      allow_any_instance_of(ActionView::Base).to receive(:redesign_enabled?).and_return(redesign_enabled)
```

Comment:
Could we move this to a shared context that is included in the target spec as this seems to be repeated across multiple specs?